lib/srv/db/cloud/watchers/watcher_test.go

```diff
@@ -259,6 +309,34 @@ func makeRDSClusterWithExtraEndpoints(t *testing.T, name, region string, labels
 	return cluster, append(types.Databases{primaryDatabase, readerDatabase}, customDatabases...)
 }
 
+// rdsInstanceStatus returns an option function for makeRDSInstance to overwrite status.
+func rdsInstanceStatus(status string) func(*rds.DBInstance) {
```

Comment:
nit: can we follow the `with..` func naming convention for optimal param ? 
```suggestion
func withRDSInstanceStatus(status string) func(*rds.DBInstance) {
```